src/mlpack/methods/decision_tree/decision_tree.hpp

```diff
@@ -529,6 +589,15 @@ using DecisionStump = DecisionTree<FitnessFunction,
                                    ElemType,
                                    false>;
 
+/**
+ * Convenience typedef for decision tree (Three level decision trees).
```

Comment:
It's called `ID3DecisionStump` not because it has 3 levels but because it's a stump (one level) made with the ID3 algorithm. :)

Reply:
Sorry for this I changed it .